tools/uvisor-tests.txt

```diff
@@ -1 +1 @@
-0ac4c38729d2602bff96d68983a7790143210912
\ No newline at end of file
+19ac0f1047d9bb7223fb8854f531968a8fe55d7b
```

Comment:
Is it necessary to update uvisor-test.txt? What modifications to uvisor-test do you need to support `UVISOR_STATIC_ASSERT`?